.github/workflows/autobuild.yml

```diff
@@ -65,100 +88,140 @@ jobs:
   release_assets:
     name:                           Build assets for ${{ matrix.config.config_name }}
     needs:                          create_release
-    runs-on:                        ${{ matrix.config.building_on_os }}
     strategy:
       fail-fast:                    false
       matrix: # Think of this like a foreach loop. Basically runs the steps with every combination of the contents of this. More info: https://docs.github.com/en/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
         config:
-           - config_name:           Linux (normal)
-             target_os:             linux
-             building_on_os:        ubuntu-18.04
-             asset_path:            deploy/Jamulus_amd64.deb
-             asset_name:            jamulus_${{ needs.create_release.outputs.version_name }}_ubuntu_amd64.deb
-           - config_name:           Linux (headless)
+           - config_name:           AndroidAPK (artifact+codeQL)
+             target_os:             android
+             building_on_os:        ubuntu-20.04
+             cmd1_prebuild:         "sh ./autobuild/android/autobuild_apk_1_prepare.sh "
```

Comment:
Question: Why did you add a space after the quote here? (applies to multiple other commands) below.

Reply:
Done

Reply:
you mean the offset of the last quote?
Historical, the jamulus_project_path used to be passed as a "normal" parameter to the call, before I decided that a named environment variable will be more intuitive to read for others. Also more flexible if additional parameters are needed in some distant future.

Reply:
> you mean the offset of the last quote?

Yes. 

I don't see any benefit from adding a space here... I'd remove it, if possible. You can always add a space here.

Reply:
Ok. Thanks